awx/ui/src/components/RoutedTabs/RoutedTabs.js

```diff
@@ -39,7 +43,7 @@ function RoutedTabs({ tabsArray }) {
           aria-label={typeof tab.name === 'string' ? tab.name : null}
           eventKey={tab.id}
           key={tab.id}
-          link={tab.link}
+          href={`#${tab.link}`}
```

Comment:
This is a bit quirky... help the URL appear correctly while mousing over the link.

Ideally, I'd like to remove the hash from URLs entirely to make this unnecessary, but it may be too late for that